frontend/pages/entities/index.jsx

```diff
@@ -62,7 +63,7 @@ export default function Entities({ entitiesData }) {
     <Grid item key={entity['pk']}
       xs={6} sm={4} md={3}>
       <a href={`entities/${entity['pk']}`}>
-        <Card>
+        <Card elevation={3}>
```

Comment:
I don't have a strong preference, but I wonder if we should make our own card component (extending MUI's component so that we can specify `elevation` (and anything else) in a single place.

Reply:
I think we are very likely to do this eventually. As long as we're mostly using MUI, the best way to accomplish this will probably be with [MUI's theming](https://material-ui.com/customization/theming/) and [overriding default props](https://material-ui.com/customization/globals/).